test/e2e/logforwarding/fluent/fluent_test.go

```diff
@@ -92,7 +92,7 @@ var _ = Describe("[ClusterLogForwarder]", func() {
 			for _, name := range logTypes {
 				name := name // Don't bind to range variable
 				r := f.Receiver.Sources[name].TailReader()
-				Expect(r.ReadLine()).To(SatisfyAny(Equal(""), ContainSubstring(fmt.Sprintf(`"log-type":%q`, name))))
+				Expect(r.ReadLine()).To(SatisfyAny(Equal(""), ContainSubstring(fmt.Sprintf(`"log_type":%q`, name))))
```

Comment:
receivers are vector-only, we shouldn't need to change anything fluent-specific...

Reply:
It was failing on one of the e2e tests, but it was maybe something temporary, reversed the change and see what happens